apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -396,7 +397,7 @@ def parse(element_html, data):
     # a failure, it would be due to corrupt data from the hidden input element).
     try:
         a_sub = b64_to_dict(a_sub)
-    except Exception:
+    except (binascii.Error, ValueError, TypeError):
```

Comment:
These are the only 3 errors possible from this call chain.

Reply:
🤔 I'm still not convinced of the value of changes like this. Why would we not want to recover from _any_ error that might be encountered? And in general it seems really hard to reason about what exceptions a particular function call could produce.

Reply:
I would prefer to have two exception clauses here: one for known errors and one for unknown errors. This lets us distinguish between errors we expect to happen vs errors that don't.

Reply:
Why does it matter if the action we'll take doesn't change between those two cases? Or are you saying that it should?

Reply:
I'm saying it should -- one is errors we aren't aware of, and likely means something unexpected happened.